src/content/docs/linter/index.mdx

```diff
@@ -377,6 +431,37 @@ If you are migrating from ESLint, we have a dedicated [migration guide](/guides/
    The command will suppress all linting violation that Biome finds, using the reason `"suppressed due to migration"`. Now the linter shouldn't error anymore, and it's possible to remove the suppression comments at a later stage.
 </Steps>
 
+
+## Linter Groups
+
+The linter divides rules under _groups_. Groups are meant to offer some sort of category which rules falls under. This information becomes useful, for users, when choosing a rule to enable/disable, or for developers when creating new lint rules.
+
+### Accessibility
+
+Rules focused on preventing accessibility problems.
+
+### Complexity
+Rules that focus on inspecting complex code that could be simplified.
+
+### Correctness
+Rules that detect code that is guaranteed to be incorrect or useless.
+
+### Nursery
+New rules that are still under development.  Nursery rules require explicit opt-in via configuration on stable versions because they may still have bugs or performance problems. They are enabled by default on nightly builds, but as they are unstable their diagnostic severity may be set to either error or warning, depending on whether we intend for the rule to be recommended or not when it eventually gets stabilized. Nursery rules get promoted to other groups once they become stable or may be removed.  Rules that belong to this group are not subject to semantic version.
+
+### Performance
+Rules catching ways your code could be written to run faster, or generally be more efficient.
+
+### Security
+Rules that detect potential security flaws.
+
+### Style
+Rules enforcing a consistent and idiomatic way of writing your code. This rules, by design, *will never emit errors* unless configured otherwise.
```

Comment:
```suggestion
Rules enforcing a consistent and idiomatic way of writing your code. By default, these rules will only generate warnings instead of errors.
```